src/Api/API.php

```diff
@@ -3325,6 +3325,15 @@ protected function applyMassiveAction(
         $action = explode(':', $action_key);
         $processor = $action[0];
 
+        $params['field'] = array_keys($params)[0];
+
+        foreach (Search::getOptions($itemtype) as $key => $value) {
+            if ($value['field'] == $params['field']) {
```

Comment:
Search options may have the `'massiveaction' => false` property that should prevent the field to be updatable using massive actions.
```suggestion
            if ($value['field'] == $params['field'] && ($value['massiveaction'] ?? true) !== false) {
```